.travis.yml

```
@@ -9,18 +9,16 @@ d:
   # this way the overall test time gets cut down (GDC/LDC are a lot
   # slower tham DMD, so they should be started early), while still
   # catching most DMD version related build failures early
-  - dmd-2.088.0
+  - dmd-2.090.1
   - dmd-2.078.3
-  - ldc-1.17.0
-  - ldc-1.16.0
+  - ldc-1.20.0
```

```suggestion
  - ldc-1.20.1
```

(for attempting to fix the CI error and to retrigger with the changed CI configuration)